js/internal/modules/modules.go

```diff
@@ -61,3 +61,42 @@ func Register(name string, mod interface{}) {
 	}
 	modules[name] = mod
 }
+
+// GetJSModules returns a map of all js modules
+func GetJSModules() map[string]HasModuleInstancePerVU {
```

Comment:
I don't understand why we need to wrap all modules artificially in a `HasModuleInstancePerVU` implementation, when they explicitly don't have a different instance for each VU :confused: In fact, I think this adds confusion without actually adding any benefits... 

The only "benefit" I can see is that is that we don't use `map[string]interface{}` here, but rather hide the "nasty" `interface{}` behind a useless `HasModuleInstancePerVUDummyWrapper`, which seems pointless.

Reply:
My reason was that GetJSModules can be used by other things beside the current use in `requireModule` and in all of those cases this will need to be checked for each module. 

I also expected that such a thing will happen while I was doing the rest of the code changes, but at least at the time of writing apparently I didn't need to use it directly (or at least nowhere where this changes broke the code was this necessary, and I haven't gone through all the tests to see if a good use of this can be found there). 

Given that I am fine with reverting to the less types variant

Reply:
If it happens, we can always make a short helper function that handles it, so I'd definitely prefer if we don't have a mandatory `HasModuleInstancePerVUDummyWrapper` when we ain't gonna need it :smile: